<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoSteering</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GSPrototype.qrc">
    <normaloff>:/Resource/image/App.ico</normaloff>:/Resource/image/App.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Project"/>
    <addaction name="action_Open_Project"/>
    <addaction name="separator"/>
    <addaction name="action_Save_Project"/>
    <addaction name="action_Save_Project_As"/>
    <addaction name="separator"/>
    <addaction name="action_Close_Project"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDataConvert">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="action_Close_All"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="action_Cascade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionNew_2D_window"/>
    <addaction name="actionNew_3D_window"/>
    <addaction name="separator"/>
    <addaction name="action_Modeling"/>
    <addaction name="action_Curve_Contrast"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuDataConvert"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExpand_All">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapsed_All">
   <property name="text">
    <string>Collapsed All</string>
   </property>
  </action>
  <action name="actionNew_2D_window">
   <property name="text">
    <string>New &amp;2D window</string>
   </property>
  </action>
  <action name="actionNew_3D_window">
   <property name="text">
    <string>New &amp;3D window</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Cl&amp;ose</string>
   </property>
  </action>
  <action name="action_Close_All">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="text">
    <string>Ne&amp;xt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="action_New_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="action_Open_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="action_Save_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="action_Save_Project_As">
   <property name="text">
    <string>Save Project As ...</string>
   </property>
  </action>
  <action name="action_Close_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Modeling">
   <property name="text">
    <string>Modeling</string>
   </property>
  </action>
  <action name="action_Curve_Contrast">
   <property name="text">
    <string>Curves Contrast</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GSPrototype.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Tile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Next</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activateNextSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Previous</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activatePreviousSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeActiveSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close_All</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
